Assets/Scripts/Gestures.cs

```diff
@@ -0,0 +1,113 @@
+﻿using Mapbox.Examples;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Gestures : MonoBehaviour
+{
+    private GameObject target;
+    Camera cam;
+    [SerializeField, Range(0, 3)] private float rotationRate = 0.4f;
+    [SerializeField] GameObject compassImage;
+    [SerializeField] GameObject textWarning;
+    private bool firstWarning = true;
```

Comment:
Nit: I usually like using something that sounds like it can translate to a bool for boolean values like `hasShownFirstWarning` or something...